docs/components/user/user-button.mdx

```diff
@@ -40,18 +40,20 @@ In this example, `<UserButton />` is mounted inside a header component, which is
               <SignInButton />
             </SignedOut>
           </header>
-      );
+        );
       }
 
-      function MyApp({ pageProps }) {
+      export default function RootLayout({ children }: { children: React.ReactNode }) {
         return (
-          <ClerkProvider {...pageProps}>
-            <Header />
-          </ClerkProvider>
+          <html lang="en">
+            <ClerkProvider>
+              <Header />
```

Comment:
Layouts need to render their children, which will be the matching page component.

```suggestion
              <Header />
              {children}
```

Reply:
great catch!!